docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>some of the text still references ballooning stability</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Resolved in #1290